FullStoryMiddlewareTests/FullStoryMiddlewareTest.m

```diff
@@ -209,6 +209,18 @@ - (void)testFullStoryMiddleware_IdentifyPayload_UserTraits_FSIdentifyCalled {
     OCMVerify([fs identify: @"testUserId" userVars:@{@"firstName_str": @"test first"}];);
 }
 
+- (void)testFullStoryMiddleware_IdentifyPayload_DisableIdentifyEvents_FSIdentifyNotCalled {
+    // create a mock for the FS
+    id fs = OCMClassMock([FS class]);
```

Comment:
Nit: just "FS", not "the FS". Or write "the FS class".

I also think this comment isn't necessary since it's just retelling exactly what the code below it is doing. It doesn't explain anything or add extra information.